apps/event/mutations.py

```diff
@@ -551,6 +555,12 @@ def mutate(root, info, event_id):
         return SignOffEvent(result=event, errors=None, ok=True)
 
 
+class ExportContextOfViolence(ExportBaseMutation):
```

Comment:
Let's use plural for consistency.
```suggestion
class ExportContextOfViolences(ExportBaseMutation):
```